### ISSUE TITLE
A11y_SqlDatabaseProjectsExtension_DatabaseProject_ScreenReader: ORCA Screen reader announcing name property of "New, Open existing, refresh buttons"

### ISSUE THREAD

**Issue by PoojaNamde** [microsoft/azuredatastudio#23547]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

### Git hub Tags:
#A11yTCS; #A11yMAS; #A11ySev2; #DesktopApp; #Benchmark; #ORCA; #WCAG4.1.2; #FTP; #SH-SQL Azure Data Studio-Linux-Jul23; #SQL Azure Data Studio; #Linux;

### Environment Details:
Application Name: Sql Database Projects Extension
Operating System: Ubuntu 20.04.2 LTS
Screen Reader: ORCA

### Repro Steps:
1. Launch Azure Data Studio application.
2. TAB to 'Extension' and search for “SQL Database Projects” and install it.
3. TAB to "Database project" and press ENTER key.
4. Start ORCA screen reader.
5. TAB to "New, Open existing, refresh buttons" and observe the issue.

### Actual Result: 
ORCA Screen reader announcing name property of "New, Open existing, refresh buttons".

### Expected Result:
ORCA Screen reader should announce name property of "New, Open existing, refresh buttons".

### User Impact:
The screen reader users will not be able to know the purpose of the buttons as it is not announcing their names and will miss the information.

### Recommendations:
Refer [Repository of bug fixes code snippets](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2faka.ms%2faccsnippet&data=05%7c01%7cv-samhithaks%40microsoft.com%7c535b4a834ef843acdfb508da53769f91%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637914066217264155%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=xoIfLB9Iq6HVIPyUc42rS%2BlVQCt9m%2BaUokp32Kaz1nE%3D&reserved=0).

### Attachment:
[23547_Issue.webm](https://github.com/microsoft/azuredatastudio/assets/90901519/80530faa-4e64-4f32-bdb0-2c64f4e344d8)

**"Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility or the Divisional Driver"**

**Comment by kisantia** [comment]
@PoojaNamde This is already fixed. Please try the [latest insiders build](https://github.com/microsoft/azuredatastudio#try-out-the-latest-insiders-build-from-main-branch).

**Comment by PoojaNamde** [comment]
#Closed; #Regressed:07-03-23;

Verified this issue in the latest Azure Data Studio- Insiders.
Version: 1.45.0-Insider
Issue is fixed, hence closing the bug.
